uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1018.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1018 {
+
+  /**
+   * Make the is Leave Approver of/has Leave Approved by Relationship
```

Comment:
This and the other upgrader description might be a bit more readable if the relationship type was in quotes